apps/web/app/api/(internal)/insights/lib/document.ts

```
@@ -14,10 +14,15 @@ import {
 } from "@formbricks/types/documents";
 import { DatabaseError } from "@formbricks/types/errors";
 
+export type TCreatedDocument = TDocument & {
+  isSpam: Boolean;
```

Biome

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)